src/main/webapp/app/detail-overview-list/components/programming-auxiliary-repository-buttons-detail.component.html

```diff
@@ -0,0 +1,25 @@
+<ul>
+    @for (auxiliaryRepository of detail.data.auxiliaryRepositories; track auxiliaryRepository) {
+        @if (auxiliaryRepository.id && auxiliaryRepository.repositoryUri && detail.data.exerciseId) {
+            <li class="mt-2">
+                <span>Repository: {{ auxiliaryRepository.name }}</span>
+                <jhi-code-button class="ms-2" [smallButtons]="true" [repositoryUri]="auxiliaryRepository.repositoryUri" />
```

Comment:
How are authentication and credentials handled for the links of the auxiliary repos? Are both SSH and HTTPS available/shown?

From an aesthetics point of view: Would it make sense to list the auxiliary repos like the ones above? I feel like the variable button indentation depending on the repo name, looks a bit wonky

Reply:
> 1. How are authentication and credentials handled for the links of the auxiliary repos? Are both SSH and HTTPS available/shown?

I am not sure if I do understand the question properly - we are reusing the code button component, so SSH and HTTPS will be available
![image](https://github.com/user-attachments/assets/d529170a-d42b-493f-a617-7ef54ad457f6)

> 2. From an aesthetics point of view: Would it make sense to list the auxiliary repos like the ones above? I feel like the variable button indentation depending on the repo name, looks a bit wonky

I agree that the current solution very aesthetic - I think that the best solution would be to come up with a good solution to merge the `Checkout Directory` information with the repository buttons. As this PR addresses the code quality perspective I will not address it here, but I appreciate the input 😄 
![image](https://github.com/user-attachments/assets/144ceec2-b873-4497-aa0e-b3144977fe0f)

Reply:
I have added an issue, so the input will not be forgotten [Programming exercises: Improve visual appearance of repository information in exercise details view #9177](https://github.com/ls1intum/Artemis/issues/9177)